src/constants.ts

```diff
@@ -30,6 +30,8 @@ const RELAY_CHAIN_URL = process.env.REACT_APP_RELAY_CHAIN_URL;
 const DEFAULT_ACCOUNT_SEED = process.env.REACT_APP_DEFAULT_ACCOUNT_SEED;
 const FAUCET_URL = process.env.REACT_APP_FAUCET_URL || 'http://localhost:3035';
 
+const PRICES_URL = `https://api.coingecko.com/api/v3/simple/price?ids=bitcoin,kintsugi,kintsugi-btc,kusama,polkadot,interlay,interlay-btc&vs_currencies=usd`;
```

Comment:
Are we considering creating a service for prices in the future?

Reply:
Not that I'm aware of - they're not that important really. We use them to calculate approx. dollar amounts when users are transferring etc. but they're not used for anything critical.

Reply:
that's why we don't really care that much if it fails - only effect is that dollar amounts aren't calculated.